src/apps/common/ns_turn_utils.c

```diff
@@ -526,7 +524,15 @@ void err(int eval, const char *format, ...) {
 }
 #endif
 
-void turn_log_func_default(char *file, int line, TURN_LOG_LEVEL level, const char *format, ...) {
+#if !defined(HAS_LOG4CPLUS)
+void *turn_log_init() { return NULL; }
+
+void turn_log_clean(void *log) {}
+
+int turn_log_set_conf_file(const char *file) { return 0; }
+
+void turn_log_func_default(const char *file, int line, const char *f, char *category, TURN_LOG_LEVEL level,
```

Comment:
This appears to change the function signature and i'm not sure that all possible uses of it were converted over?

Reply:
I am sure!